client/internal/storage/repository/secret.go

```diff
@@ -0,0 +1,97 @@
+package repository
+
+import (
+	"client/internal/logger"
+	"client/internal/models"
+	"context"
+	"log/slog"
+)
+
+const upsertSQL = `
+		INSERT INTO secrets (id, name, pass, meta, user_id, version)
+		VALUES ($1, $2, $3, $4, $5, $6)
+		ON CONFLICT ( id ) DO UPDATE SET
+		name = excluded.name,
+		pass = excluded.pass,
+		meta = excluded.meta,
+		version = excluded.version
+`
+
+const listSQL = `
+	SELECT id, name, pass, meta, version 
+	FROM secrets 
+	WHERE user_id = $1 AND version > $2
+`
+const showSQL = "SELECT id, name, pass, meta, version from secrets where ID = $1"
+
```

Comment:
Мне кажется так читается лучше.
```suggestion
const (
	upsertSQL = `
		INSERT INTO secrets (id, name, pass, meta, user_id, version)
		VALUES ($1, $2, $3, $4, $5, $6)
		ON CONFLICT ( id ) DO UPDATE SET
		name = excluded.name,
		pass = excluded.pass,
		meta = excluded.meta,
		version = excluded.version
	`

	listSQL = `
		SELECT id, name, pass, meta, version 
		FROM secrets 
		WHERE user_id = $1 AND version > $2
	`

	showSQL = "SELECT id, name, pass, meta, version from secrets where ID = $1"
)

```

Reply:
Согласен. Поправил в сервере и в клиенте